client/app/license/order/license-order.html

```diff
@@ -264,8 +263,7 @@ <h2 data-translate="license_order_step4_title"></h2>
                     </div>
                 </div>
 
-                <div data-ng-if="!agoraEnabled"
-                     data-ng-show="order.url">
+                <div data-ng-if="!agoraEnabled && order.url">
                     <div class="row">
                         <div class="col-md-9">
```

Comment:
Not yours but, these classes (`row`and `col-md-9`) are now not necessary and then can be removed, since you removed the `col-md-3`, to place the button under the order URL.
(This will avoid to change this display format in a second time ;) )